### PR TITLE
[enhancement](cloud) update rows and versions after load

### PR DESCRIPTION
**Pull request by freemandealer**
This commit doing the following after load:
1. report number of updated rows to AnalysisMgr
2. inform AnalysisManager when partition doing first
3. update cached version in CloudPartition

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by freemandealer** [comment]
run buildall

**Comment by freemandealer** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java [hunk 1]
```diff
@@ -3665,6 +3670,85 @@ private TGetBackendMetaResult getBackendMetaImpl(TGetBackendMetaRequest request,
         }
     }
 
+    class TableStats {
+        public long updatedRowCount;
+    }
+
+    public TStatus reportCommitTxnResult(TReportCommitTxnResultRequest request) throws TException {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java:3677]: This method may be not the latest version.

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Comment by freemandealer** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Comment by freemandealer** [comment]
run buildall

**Comment by freemandealer** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by freemandealer** [comment]
run p0

**Pull request merged**